commerce_coordinator/apps/core/cache.py

```diff
@@ -30,3 +33,61 @@ def get_processing_payment_state_cache_key(payment_number):
     Wrapper method on edx_django_utils to processing_payment_state_cache.
     """
     return get_payment_state_cache_key(payment_number=payment_number, payment_state=CachePaymentStates.PROCESSING.value)
+
+
+def set_payment_processing_cache(payment):
+    """
+    Utility for setting cache for the payment object in PROCESSING Cache.
+    """
+    cached_payment_serializer = CachedPaymentSerializer(data=payment)
```

Comment:
Added CachedPaymentSerializer in all our caches to make sure caches are consistent